.idea/.idea.URDF-Importer.dir/.idea/projectSettingsUpdater.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
no need to check in the Rider related files. you can add `.idea` to the gitignore file

Reply:
Thank you.
I add remove files under .idea directory and add to .gitignore.

Reply:
@hakuturu583 , could you double check if you have pushed your latest commit? thanks.

Reply:
Thank you!
I fixed.
Please check.